embedmanager/embedmanager.py

```diff
@@ -335,32 +335,39 @@ async def embed_info(self, ctx: commands.Context, name: StoredEmbedConverter):
 
     @embed_group.group(name="edit", usage="<option>", invoke_without_command=True)
     @checks.has_permissions(PermissionLevel.MODERATOR)
-    async def embed_edit(
-        self,
-        ctx: commands.Context,
-        message: BotMessage,
-        color: Optional[discord.Color],
-        title: str,
-        *,
-        description: str,
-    ):
+    async def embed_edit(self, ctx: commands.Context, message: BotMessage, index: int = 0):
         """
-        Edit a message sent by Bot's embeds.
+        Edit a message's embed sent by the bot.
```

Comment:
```suggestion
        Edit a message's embed sent by the bot.
        This will initiate the Embed Editor panel with interactive buttons and text inputs session.
        The values for the input fields are pre-defined based on the source embed.
```